omnigibson/macros.py

```diff
@@ -123,7 +123,7 @@ def determine_gm_path(default_path, env_var_name):
 gm.ENABLE_OBJECT_STATES = True
 
 # Whether to enable transition rules or not
-gm.ENABLE_TRANSITION_RULES = True
+gm.ENABLE_TRANSITION_RULES = False
```

Comment:
Undo

Reply:
We haven't fixed transition rules on `og-develop` though. Do we want to keep this False for now?

Reply:
Why is this showing as a diff then? Did we not merge og-develop here?